src/AdminConsole/Pages/Account/Profile.cshtml

```diff
@@ -8,8 +8,9 @@
     var requestToken = Antiforgery.GetAndStoreTokens(Model.HttpContext).RequestToken;
 }
 
-<panel header="Passkeys">
+<div class="panel">
```

Comment:
Are we moving away from Panel?

Reply:
You cannot use razor components inside tag helpers